alicloud/provider.go

```diff
@@ -125,6 +125,7 @@ func Provider() terraform.ResourceProvider {
 			"alicloud_eips":                   dataSourceAlicloudEips(),
 			"alicloud_key_pairs":              dataSourceAlicloudKeyPairs(),
 			"alicloud_kms_keys":               dataSourceAlicloudKmsKeys(),
+			"alicloud_dns_support_lines":      dataSourceAlicloudDnsSupportLines(),
```

Comment:
please use `alicloud_dns_resolution_line` instead of `alicloud_dns_support_lines `